oauth2_http/clirr-ignored-differences.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- see http://www.mojohaus.org/clirr-maven-plugin/examples/ignored-differences.html -->
+<differences>
+  <difference>
+    <differenceType>6001</differenceType>
+    <className>com/google/auth/oauth2/ExternalAccountCredentials$Builder</className>
```

Comment:
why we need to do this?

Reply:
`protected quotaProjectId` was removed from builder of external account creds because a `protected quotaProjectId` was added to its base class. Got a build error saying change in public interface but we really did not change. Same thing is available from parent. So adding exception,

Reply:
Got it, i was not sure why it is only that credential. Looks good.